src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+const thead = document.querySelectorAll('thead tr th');
+const tbody = document.querySelector('tbody');
+const tbodyArray = Array.from(tbody.querySelectorAll('tr'));
+const sortStates = [];
+
+thead.forEach((item) => {
+  sortStates[item] = 'asc';
```

Comment:
The `sortStates` array is being indexed with DOM elements (`item`), which is not valid. Consider using numerical indices or keys that correspond to the column index.